wkcuber/convert_nifti.py

```diff
@@ -67,13 +67,6 @@ def create_parser() -> ArgumentParser:
         default=None,
     )
 
-    parser.add_argument(
-        "--write_tiff",
-        help="Output tiff dataset instead of wkw.",
-        default=False,
-        action="store_true",
-    )
```

Comment:
Are we sure that this is not used? @leowe You are probably the one who can decide best whether this parameter can be removed.

Reply:
I think this can be removed. I used it only for debug purposes and as far as I know we currently have no direct use case for it in a project context.